packages/common/src/store/pages/chat/slice.ts

```diff
@@ -0,0 +1,96 @@
+import type { TypedCommsResponse, UserChat, ChatMessage } from '@audius/sdk'
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+
+import { Status } from 'models'
+
+type ChatState = {
```

Comment:
i typically export this state, and use it for the types, rather than the `ReturnType<typeof reducer>` pattern